src/services/MailService.js

```diff
@@ -1,7 +1,9 @@
 import {createTransport} from 'nodemailer'
-import smptConfig from '../smptConfig'
+import smtpConfig from '../smtpConfig'
 import {compileFile} from 'pug'
 
+const develop = true
```

Comment:
This should be a constant which can be used in the whole app and also settable by command line. Usually node probgramms use the environment variable `NODE_ENV=dev` for this.

Reply:
You are right, I was just looking for a quick way to prevent emails from being sent :)

Reply:
Maybe that's a thing that should be done in another task, I'll create one.